tools/coverage/Makefile

```diff
@@ -5,9 +5,20 @@ REGISTRY ?= gcr.io
 PROJECT  ?= knative-tests
 PUSH     ?= docker push
 
-coverage-image:
+# Making dev version of coverage-image
+coverage-image-dev:
```

Comment:
there seems to be a lot of common stuff between the 2 images. can we make a common section and call that from each of it?

Reply:
+1. Refactor these into build and publish steps. Looking forward to the day that we don't need a separate job for the dev image.

Reply:
Is there any example of a separate build and publish script? I guess we already have it for some other images?

Reply:
See the one for prow-tests